samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/OktaVerify.java

```diff
@@ -160,4 +160,17 @@ public void she_sees_a_page_with_qr_code_displayed_for_scanning() {
 		oktaVerifyPage.waitForWebElementDisplayed(oktaVerifyPage.qrCodeDisplayed);
 		Assert.assertTrue(oktaVerifyPage.qrCodeDisplayed.isDisplayed());
 	}
+
+	@Then("^she is presented with an option to select Okta Verify$")
+	public void she_is_presented_with_an_option_to_select_okta_verify() {
+		Assert.assertTrue(selectAuthenticatorPage.selectAuthenticatorsForm.isDisplayed());
+		Assert.assertTrue(oktaVerifyPage.oktaVerifyOption.isDisplayed());
+
```

Comment:
This is not necessarily an issue but I would recommend removing the extra empty line here, just to keep this checkstyle compliant. Otherwise, looks good.

Reply:
Sure, I'll make sure there is no extra empty line.